pkg/apis/softwarecomposition/types_test.go

```diff
@@ -0,0 +1,285 @@
+package softwarecomposition
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_VulnerabilitySummaryMerge(t *testing.T) {
+	tests := []struct {
+		fullVulnSumm         *VulnerabilitySummary
+		vulnManifestSumm     *VulnerabilityManifestSummary
+		expectedFullVulnSumm *VulnerabilitySummary
+	}{
+		{
+			fullVulnSumm: &VulnerabilitySummary{
+				Spec: VulnerabilitySummarySpec{
+					Severities: SeveritySummary{
+						Critical: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+						High: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+						Medium: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+						Low: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+						Negligible: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+						Unknown: VulnerabilityCounters{
+							All:      10,
+							Relevant: 3,
+						},
+					},
```

Comment:
suggestion(non-blocking): you can put the creation of the object into a helper function

So that it would save you some screen space and typing